### DISCUSSION TITLE
Add document translations

### DISCUSSION THREAD

**Discussion by soramikan** [utmapp/UTM#4770] [Ideas]
It's great to have helpful documentation that can be read in a variety of languages.
Would you like to add a translation of the documentation using Jekyll's plugins such as [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin) or [Polyglot](https://github.com/untra/polyglot)?

If you do the translation, I am ready to commit.

**Comment by osy** [comment]
Unfortunately GitHub only allows a small subset of allowed plugins: https://pages.github.com/versions/ and also just the docs still has limited translation support (for templated stuff) https://github.com/just-the-docs/just-the-docs/issues/59

I haven’t looked too much into it but if you make the necessary changes to the docs repo and it works please send a PR.

**Reply by soramikan** [reply]
Thank you, I will look into it and see if I can find it. I will look into it and give it a try.
